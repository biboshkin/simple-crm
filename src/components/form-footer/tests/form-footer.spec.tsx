import React from "react";
import renderer from "react-test-renderer";
import { FormFooter } from "..";

test("renders FormFooter snapshot", () => {
  const component = renderer.create(
    <FormFooter disabled={false} loading={false} />
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
