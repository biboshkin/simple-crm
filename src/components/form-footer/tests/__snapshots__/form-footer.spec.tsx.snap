// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="ant-row ant-form-item container"
  style={Object {}}
>
  <div
    className="ant-col ant-form-item-control"
    style={Object {}}
  >
    <div
      className="ant-form-item-control-input"
    >
      <div
        className="ant-form-item-control-input-content"
      >
        <button
          className="ant-btn cancel"
          onClick={[Function]}
          type="button"
        >
          <span>
            cancel
          </span>
        </button>
        <button
          className="ant-btn submit ant-btn-primary"
          disabled={false}
          onClick={[Function]}
          type="submit"
        >
          <span>
            submit
          </span>
        </button>
      </div>
    </div>
  </div>
</div>
`;
